src/SignalR/common/Http.Connections/src/ServerSentEventsMessageFormatter.cs

```diff
@@ -11,8 +11,8 @@ namespace Microsoft.AspNetCore.Http.Connections
 {
     internal static class ServerSentEventsMessageFormatter
     {
-        private static readonly byte[] DataPrefix = { (byte)'d', (byte)'a', (byte)'t', (byte)'a', (byte)':', (byte)' ' };
-        private static readonly byte[] Newline = { (byte)'\r', (byte)'\n' };
+        private static readonly ReadOnlyMemory<byte> DataPrefix = new[] { (byte)'d', (byte)'a', (byte)'t', (byte)'a', (byte)':', (byte)' ' };
```

Comment:
I think the nice optimization with this only works for `private static ReadOnlySpan<byte> blah => new byte[] {...};`

Reply:
Sorry, I didn't follow - are you recommending changing this to a ReadOnlySpan and calling `AsMemory` at the callsite?

Reply:
Yeah
https://github.com/dotnet/aspnetcore/blob/4dc663a0d46467908f5463d5004b0ae574648f4a/src/Middleware/WebSockets/src/HandshakeHelpers.cs#L19-L20

Reply:
Unfortunately we can't go from Span -> Memory. I'll let this be

Reply:
@BrennanConroy Why is this writing to HttpResponse.Body instead of BodyWriter. Then we could use the optimization you suggested.

Reply:
Because BodyWriter didn't exist when this code was written (plus it's SSE so...)